client/src/App.js

```diff
@@ -1,25 +1,11 @@
 import React from 'react';
 import logo from './logo.svg';
+import Register from './Register.js'
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Register></Register>
```

Comment:
nit: turn this into a self closing tag, i.e. `<Register />`, if the tag has no children (nothing between the `<Register>` and `</Register>`)